<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScraperSettingsTable</class>
 <widget class="QWidget" name="ScraperSettingsTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>439</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="scraperList">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkEnableAdultScrapers">
       <property name="text">
        <string>Enable adult scrapers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="scraperDetailPages"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
